core/domain/email_manager_test.py

```diff
@@ -190,6 +190,7 @@ def test_correct_rights_are_written_in_manager_role_email_body(self):
             '<ul>'
             '<li>Change the exploration permissions</li><br>'
             '<li>Edit the exploration</li><br>'
+            '<li>Translate the exploration</li><br>'
```

Comment:
I don't think you need this in the non-translator emails. It comes under "edit".

Reply:
Done!